setup.py

```diff
@@ -51,8 +51,9 @@ def run(self):
         errTwo = call(self.doc_command)
         sys.exit(errOne or errTwo)
 
+
 # Generate API documentation only if we are running on readthedocs.io
-on_rtd = getenv('READTHEDOCS', None) != None
+on_rtd = True if getenv('READTHEDOCS', None) is not None else False
```

Comment:
I don't see how the `True if ... else False` helps us, that's just overly verbose. The change from `!=` to using `is` makes sense